src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -35,6 +38,9 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isGiverTeam = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
         isRecipientTeam = questionAttributes.recipientType.isTeam();
+        isRecipientOwnTeamMembers = questionAttributes.recipientType.isOwnTeamMembers();
+        isRecipientOtherTeams = questionAttributes.recipientType.isOtherTeams();
```

Comment:
Hmm, not sure if I follow. This field is independent of what possible recipients there already are, right?